paper-progress.html

```diff
@@ -10,7 +10,6 @@
 
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../paper-styles/paper-styles.html">
-<link rel="import" href="../paper-styles/paper-styles-classes.html">
```

Comment:
`paper-progress` uses only the color in paper-styles, you can also just import `paper-styles/color.html`.

Also we need to import `iron-flex-layout/iron-flex-layout.html` for using custom property `--layout-fit`.